js/views/aliases-list.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * ownCloud - Mail
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ * @copyright Tahaa Karim 2016
+ */
+
+define(function(require) {
+	'use strict';
+
+	var $ = require('jquery');
+	var Marionette = require('marionette');
+	var Handlebars = require('handlebars');
+	var AliasesListTemplate = require('text!templates/aliases-list.html');
+	var AliasesController = require('controller/aliasescontroller');
+
+	return Marionette.CompositeView.extend({
+		template: Handlebars.compile(AliasesListTemplate),
+		templateHelpers: function() {
+			var aliases = this.aliases;
+			return {
+				aliases: aliases
+			};
+		},
+		ui: {
+			'deleteButton' : 'button'
+		},
+		events: {
+			'click @ui.deleteButton': 'deleteAlias'
+		},
+		aliases: null,
+		currentAccount: null,
+		initialize: function(options) {
+			this.currentAccount = options.currentAccount;
+			this.aliases = this.currentAccount.get('aliases');
+		},
+		deleteAlias: function() {
```

Comment:
maybe have a look how we render other collection data like accounts. There we have a view for the collection and child views for the single accounts. Those ChildViews encapsulate events too, hence you won't need magic like
```
var target = $(event.target);
target.removeClass('icon-delete').addClass('icon-loading-small');
var aliasId = target.val();
```